supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -33,6 +55,17 @@
 
 *See also*:
 
+[discrete]
+[[NIC]]
```

Comment:
The anchor should be lowercase "nic". Can you also update the xref that uses it when you change it?

Reply:
done